tests/CppUTest/TestRegistryTest.cpp

```diff
@@ -425,3 +430,16 @@ TEST(TestRegistry, reverseZeroTests)
 
     CHECK(NULLPTR == myRegistry->getFirstTest());
 }
+
+TEST(TestRegistry, doesNotCrashIfNotSetToCrash)
+{
+    CHECK_FALSE(MockTest::wouldCrash());
+}
+
+TEST(TestRegistry, crashesIfSetToCrash)
+{
+    myRegistry->setCrashOnFail();
+
+    CHECK(MockTest::wouldCrash());
```

Comment:
Why not use the "static void setCrashMethod(void (*crashme)());" instead of exposing the test to the inner state?

Reply:
Actually, when I remove the Registry from the chain, I don't need these tests and I can revert the Utest.{cpp,h} changes anyway!